extract_msg/named.py

```diff
@@ -64,8 +64,11 @@ def __init__(self, msg):
             self.__guids = guids
 
             for entry in entries:
-                streamID = properHex(0x8000 + entry['pid'])
-                self.__properties.append(StringNamedProperty(entry, names[entry['id']]) if entry['pkind'] == NamedPropertyType.STRING_NAMED else NumericalNamedProperty(entry))
+                if entry['id'] in names:
+                    streamID = properHex(0x8000 + entry['pid'])
```

Comment:
Not directly related to the issue, but looks like this line could be removed since `streamID` is unused.

Reply:
Gonna take me a bit to look over the changes and what might have went wrong in the code to see what the best option is. Meanwhile, can you tell me how you did this thing that highlighted a section of code?

Reply:
@TheElementalOfDestruction Sounds good. I used this hex editor: https://hexfiend.com

Reply:
To be clear, I meant the highlighted section of code in the pull request I am responding to. Some kind of code suggestion?

Reply:
Also this suggestion is 100% the correct course. Pretty sure that was just something that got missed when I changed that section of code, so that line can just be deleted

Reply:
Oh right. I just added a comment by clicking the "+" on that line in the Files changed tab.

Reply:
Ah, thank you. Learned something new. Also sorry if you got a bunch of notifications, I was having a little trouble with using github